packages/react/src/components/button/Button.tsx

```diff
@@ -90,24 +108,31 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
       </div>
     ) : null;
 
+    const loadingOnClick: React.MouseEventHandler<HTMLButtonElement> = (event) => {
+      event.preventDefault();
+    };
+
     return (
       <button
         ref={ref}
         disabled={disabled}
+        aria-disabled={isLoading || disabled || undefined}
```

Comment:
Should the undefined be false instead?

Reply:
I guess it does not render undefined properties